apps/browser/src/platform/services/browser-script-injector.service.ts

```diff
@@ -23,9 +33,24 @@ export class BrowserScriptInjectorService extends ScriptInjectorService {
     const injectionDetails = this.buildInjectionDetails(injectDetails, file);
 
     if (BrowserApi.isManifestVersion(3)) {
-      await BrowserApi.executeScriptInTab(tabId, injectionDetails, {
-        world: mv3Details?.world ?? "ISOLATED",
-      });
+      try {
+        await BrowserApi.executeScriptInTab(tabId, injectionDetails, {
+          world: mv3Details?.world ?? "ISOLATED",
+        });
+      } catch ({ message }) {
+        // Swallow errors for host permissions, since this is believed to be a Manifest V3 Chrome bug
+        // @TODO remove when the bugged behaviour is resolved
+        if (
+          message ===
+          "Cannot access contents of the page. Extension manifest must request permission to access the respective host."
+        ) {
+          if (this.platformUtilsService.isDev()) {
```

Comment:
```suggestion
        if (
        this.platformUtilsService.isDev() &&
          message ===
          "Cannot access contents of the page. Extension manifest must request permission to access the respective host."
        ) {
```

Nit - This nested `if` statement can be eliminated

Reply:
Ah, this is meant to still throw other errors:
cc017eb473ae4861225294843353460b04d96802